hudi-hadoop-common/src/test/java/org/apache/hudi/common/table/log/block/TestHoodieDeleteBlock.java

```diff
@@ -101,9 +101,9 @@ public static Stream<Arguments> orderingValueParams() {
             {new Double[] {Double.MIN_VALUE, 0.125, 809.25, Double.MAX_VALUE}},
             {new String[] {"val1", "val2", "val3", null}},
             {new Timestamp[] {new Timestamp(1690766971000L), new Timestamp(1672536571000L)}},
-            {new LocalDate[] {LocalDate.of(2023, 1, 1), LocalDate.of(1980, 7, 1)}},
-            {new BigDecimal[] {new BigDecimal("12345678901234.2948"),
-                new BigDecimal("23456789012345.4856")}}
+            {new LocalDate[] {LocalDate.of(2023, 1, 1), LocalDate.of(1980, 7, 1)}}
+            //{new BigDecimal[] {new BigDecimal("12345678901234.2948"),
```

Comment:
this should work right? or is the scale/precision out of range? let;s fix this test.

Reply:
reverted this